packages/azure/changelog.yml

```
@@ -1,3 +1,8 @@
+- version: "1.11.5"
```

```suggestion
- version: "1.12.0"
```
You can bump minor since its an enhancement adding new field.